src/validations/employee-validation.ts

```diff
@@ -0,0 +1,39 @@
+import {User} from '../classes/user';
+import * as userServices from '../services/user-service';
+
+const errorList = {
+    nationalNumber:`A user with this national number already exists!`,
+    roleID: `Given role is not valid!`
+}
+
+const checkNationalNumber = async (nationalNumber: string): Promise<boolean> => {
+    if(!!await userServices.getUserByNationalNumber(nationalNumber))
+    {
+        return false;
+    }
+    return true;
+}
```

Comment:
"I want to validate if a national security number is correct by checking if anyone else has this same national security number" How does that make any sense?
Use a library or regex for this.

If this is function, inside the validation folder, is to check if it already exists then rename the function to indicate as such.

This also isnt a promise, just a bool return type.